test/integration_test/snapshot_restore_test.go

```diff
@@ -0,0 +1,75 @@
+// +build integrationtest
+
+package integrationtest
+
+import (
+	"testing"
+	"time"
+
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/stretchr/testify/require"
+)
+
+func testSnapshotRestore(t *testing.T) {
+	t.Run("simpleSnapshotRestoreTest", simpleSnapshotRestoreTest)
+	/*t.Run("groupSnapshotRestoreTest", groupSnapshotRestoreTest)
+	t.Run("cloudSnapshotRestoreTest", cloudSnapshotRestoreTest)
+	t.Run("groupCloudSnapshotRestoreTest", groupCloudSnapshotRestoreTest)*/
+}
+func createInPlaceRestore(t *testing.T, appKeys []string) []*scheduler.Context {
+	ctxs, err := schedulerDriver.Schedule("test",
+		scheduler.ScheduleOptions{AppKeys: appKeys})
+	require.NoError(t, err, "Error scheduling task")
+	require.Equal(t, 1, len(ctxs), "Only one task should have started")
+	return ctxs
+}
+
+func verifyInPlaceSnapshotRestore(t *testing.T, ctx []*scheduler.Context, startTime time.Time, timeout time.Duration) {
+	err := schedulerDriver.WaitForRunning(ctx[0], timeout, defaultWaitInterval)
+	// we shouldn't valid px specific things here just pass to torpedo validate inplacenspahostrestore
```

Comment:
Remove reference to PX. Mention that we shouldn't validate storage specific items.